test-cypress.sh

```diff
@@ -42,6 +46,7 @@ if [ -n "${headless-}" ] && [ -z "${pkg-}" ]; then
   yarn run test-cypress-console-headless
   yarn run test-cypress-devconsole-headless
   yarn run test-cypress-olm-headless
+  yarn run test-cypress-pipelines-headless
```

Comment:
So just to confirm if one executes  `test-prow-e2e.sh` , `test-prow-e2e.sh e2e`  or `test-prow-e2e.sh  release` then pipeline tests should be executed?  In other words, pipeline tests should be executed as part of `console` CI e2e tests.

Reply:
Yes @dtaylor113, For now I would like to integrate the pipelines test cases with console CI e2e